ppx/html.ml

```diff
@@ -15,558 +17,1762 @@ type eventType =
+  ; Attribute {name= "speed"; htmlName= "speed"; type_= String (* number |  *)}
+  ; Attribute {name= "spreadMethod"; htmlName= "spreadMethod"; type_= String}
+  ; Attribute
+      { name= "startOffset"
+      ; htmlName= "startOffset"
+      ; type_= String (* number |  *) }
+  ; Attribute
+      { name= "stdDeviation"
+      ; htmlName= "stdDeviation"
+      ; type_= String (* number |  *) }
+  ; Attribute {name= "stemh"; htmlName= "stemh"; type_= String (* number |  *)}
+  ; Attribute {name= "stemv"; htmlName= "stemv"; type_= String (* number |  *)}
+  ; Attribute
+      { name= "stitchTiles"
+      ; htmlName= "stitchTiles"
+      ; type_= String (* number |  *) }
+  ; Attribute {name= "stopColor"; htmlName= "stopColor"; type_= String}
+  ; Attribute
+      { name= "stopOpacity"
+      ; htmlName= "stopOpacity"
+      ; type_= String (* number |  *) }
+  ; Attribute
+      { name= "strikethroughPosition"
+      ; htmlName= "strikethroughPosition"
+      ; type_= String (* number |  *) }
+  ; Attribute
+      { name= "strikethroughThickness"
+      ; htmlName= "strikethroughThickness"
+      ; type_= String (* number |  *) }
+  ; Attribute {name= "String"; htmlName= "String"; type_= String (* number |  *)}
+  ; Attribute {name= "stroke"; htmlName= "stroke"; type_= String}
+  ; Attribute
+      { name= "strokeLinecap"
+      ; htmlName= "strokeLinecap"
+      ; type_= String (* type_= "butt" | "round" | "square" | "inherit" *) }
+  ; Attribute
+      { name= "strokeWidth"
+      ; htmlName= "strokeWidth"
+      ; type_= String (* number |  *) }
+  ; Attribute
+      { name= "surfaceScale"
+      ; htmlName= "surfaceScale"
+      ; type_= String (* number |  *) }
+  ; Attribute
+      { name= "tableValues"
+      ; htmlName= "tableValues"
+      ; type_= String (* number |  *) }
+  ; Attribute
+      {name= "targetX"; htmlName= "targetX"; type_= String (* number |  *)}
+  ; Attribute
+      {name= "targetY"; htmlName= "targetY"; type_= String (* number |  *)}
+  ; Attribute {name= "textAnchor"; htmlName= "textAnchor"; type_= String}
+  ; Attribute
+      {name= "textLength"; htmlName= "textLength"; type_= String (* number |  *)}
+  ; Attribute {name= "to"; htmlName= "to"; type_= String (* number |  *)}
+  ; Attribute {name= "transform"; htmlName= "transform"; type_= String}
+  ; Attribute {name= "u1"; htmlName= "u1"; type_= String (* number |  *)}
+  ; Attribute {name= "u2"; htmlName= "u2"; type_= String (* number |  *)}
+  ; Attribute
+      {name= "unicode"; htmlName= "unicode"; type_= String (* number |  *)}
+  ; Attribute
+      { name= "unicodeBidi"
+      ; htmlName= "unicodeBidi"
+      ; type_= String (* number |  *) }
+  ; Attribute
+      { name= "unicodeRange"
+      ; htmlName= "unicodeRange"
+      ; type_= String (* number |  *) }
+  ; Attribute
+      {name= "unitsPerEm"; htmlName= "unitsPerEm"; type_= String (* number |  *)}
+  ; Attribute
+      { name= "vAlphabetic"
+      ; htmlName= "vAlphabetic"
```

Comment:
Isn't this `v-alphabetic`? https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/v-alphabetic 

(also it's deprecated, not sure if it's worth including it)